subsys/bluetooth/shell/gatt.c

```diff
@@ -995,15 +995,15 @@ static int cmd_notify_mult(const struct shell *sh, size_t argc, char *argv[])
 	(void)memset(params, 0, sizeof(params));
 
 	for (uint16_t i = 0U; i < cnt_u16; i++) {
-		params[i].uuid = &vnd1_echo_uuid.uuid;
+		params[i].uuid = 0;
 		params[i].attr = vnd1_attrs;
 		params[i].data = &data;
 		params[i].len = sizeof(data);
 		params[i].func = notify_cb;
 		params[i].user_data = (void *)sh;
 	}
 
-	err = bt_gatt_notify_multiple(NULL, cnt_u16, params);
+	err = bt_gatt_notify_multiple(default_conn, cnt_u16, params);
```

Comment:
Please add a NULL-check for `default_conn` for this command as well, since we are now expecting it to be non-NULL